front/pages/w/[wId]/builder/data-sources/managed.tsx

```diff
@@ -440,20 +568,16 @@ function getTableRow({
   managedDataSource,
   isAdmin,
   isLoadingByProvider,
-  router,
   owner,
   readOnly,
+  onButtonClick,
 }: GetTableRowParams): RowData {
   const connectorProvider =
     managedDataSource.connectorProvider as ConnectorProvider;
   const isDisabled = isLoadingByProvider[connectorProvider] || !isAdmin;
 
   const buttonOnClick = () => {
-    !isDisabled
-      ? void router.push(
-          `/w/${owner.sId}/builder/data-sources/${managedDataSource.name}`
-        )
-      : null;
+    !isDisabled ? onButtonClick(managedDataSource) : null;
```

Comment:
line 591 the dataSourceUrl still points to the old route that was killed right?

Reply:
Right, we can actually get rid of `dataSourceUrl`